tests/feature/test_feature_base_dir.py

```diff
@@ -1,24 +1,33 @@
 """Test feature base dir."""
+import pkg_resources
 import pytest
 
 NOT_EXISTING_FEATURE_PATHS = [".", "/does/not/exist/"]
 
+assert_outcomes_extra = pkg_resources.get_distribution("pytest").parsed_version >= pkg_resources.parse_version("7.0.0")
```

Comment:
Note that `pkg_resources` is a third-party dependency (part of `setuptools`, but not the stdlib), and is [officially discouraged](https://setuptools.pypa.io/en/latest/pkg_resources.html):

> Use of pkg_resources is discouraged in favor of importlib.resources, importlib.metadata, and their backports (resources, metadata). Please consider using those libraries instead of pkg_resources.

Note that `importlib.metadata` also only was added in Python 3.8. However, pytest 7 added [`pytest.version_tuple`](https://docs.pytest.org/en/7.0.x/reference/reference.html#version-tuple), so you could get the version from there (and assume `False` when it does not exist).